recipes/dargs/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "dargs" %}
+{% set version = "0.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/deepmodeling/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: ec8dcbe5812964b6b346f98f8e564f6bad3ad0e0e597699eb57fbd572eecb0dd
+
+build:
+  number: 0
+  noarch: python
+  script_env:
+    - SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}
+  script: {{ PYTHON }} -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - setuptools_scm
+
+  run:
+    - python >=3.6
+
+test:
+  imports:
+    - dargs
```

Comment:
Is it worth adding `pip check` to the tests?

Reply:
added